Source/WebCore/html/ColorInputType.cpp

```diff
@@ -196,12 +196,16 @@ void ColorInputType::handleDOMActivateEvent(Event& event)
 
 void ColorInputType::showPicker() 
 {
+#if PLATFORM(IOS_FAMILY)
+    element()->focus();
```

Comment:
The showPicker spec isn't clear on if this is allowed or not (see https://github.com/whatwg/html/issues/9757) but given the spirit of the feature it feels like working rather than not working is the best solution.